publishLocally.sh

```diff
@@ -1,16 +1,24 @@
+
 # this script updates all maven projects versions (maven-java and samples) to align with the current sdk version
 # if SDK_VERSION env var is defined, it will use it, otherwise it will take the version from sbt
 # after running this script, you may run local tests or simply send a PR with the updates. 
 
 # you can pass the path to a sample to only change the version for single sample, 
 # eg: ./updatePomVersions.sh samples/java-eventsourced-counter
+# or you can pass 'all' to it and all java samples (pom.xml) will be updated
+# eg: ./updatePomVersions.sh all
```

Comment:
In need of update? since it's not ùpdatePomVersions.sh` anymore..

Reply:
Well, the whole docs were wrong. Updated now.

Reply:
indeed 👍